lib/kubernetes-deploy/kubernetes_resource/persistent_volume_claim.rb

```diff
@@ -3,12 +3,66 @@ module KubernetesDeploy
   class PersistentVolumeClaim < KubernetesResource
     TIMEOUT = 5.minutes
 
+    def init_storage_class(cache)
+      @storage_class = {}
+
+      if @definition.dig("spec", "storageClassName").nil?
+        # if no storage class is defined we try to find the default one
+        # FIXME: This assumes the DefaultStorageClass admission plugin is turned on,
+        # need a way to determine this
+        is_default_class = "storageclass.beta.kubernetes.io/is-default-class"
```

Comment:
What's the difference between the annotation you're using and `"storageclass.kubernetes.io/is-default-class"`?

Reply:
Good pick-up - I hadn't even noticed I was using `beta` (I copy and pasted from my [minikube](https://github.com/kubernetes/minikube/blob/9a7b0b91aad20b7a938cd7ba0db203864227ff7f/deploy/addons/storageclass/storageclass.yaml) deployment). I just modified my `minikube` set-up to use `storageclass.kubernetes.io/is-default-class` instead and it worked all the same. So I assume they're the same thing; and this is just a hangover from dynamic provisioning's beta days. So I guess we need support for both?

Reply:
Added support for both annotations in https://github.com/Shopify/kubernetes-deploy/pull/469/commits/4d88029021d91fc02dbb483ebd80ffbafce00c3c